### ISSUE TITLE
Suggestion: Port documentation into Github wiki

### ISSUE THREAD

**Issue by seabbs** [epiforecasts/covid-rt-estimates#107]
As the project has grown the amount of documentation has also grown (which is a very good thing). Some of this documentation targets users of the data (such as the intro to the readme and the status.csv) and some targets fairly technical devs (the remainder of the readme and the smg). 

Pushing forward I can see the amount of documentation and it's speciality growing. Given this it might make sense to switch to a more structured system. I have no strong thoughts on what this should be. GitHub wiki is attractive as it is easy to use and easy to navigate but there may be better options available (or potentially we may be better just sticking with what we have). Thoughts?

**Comment by joeHickson** [comment]
I like the readme.md as it's on the project landing page.

I also like docs with code as it handles the new feature in a branch issue nicely (.md or .rd or any one of an infinite number of choices!).

roxygen and .rd files are popular for R as a form of tech documentation as the definition is with the code but it may be overkill as this wouldn't really be used as a module. #49 type changes would improve this a lot. 

There's no easy answers for how to do documentation!

My temptation would be to say keep the .md files but update and expand - perhaps breaking down into multiple sections:
- TLDR overview (readme.md) including pointers to other .md files, published data, status pages, etc. <- focused on being a friendly landing point for non technical
- Running locally <- semi technical aimed at researchers wanting to be able to run it. 
- Developing (SMG.md) <- more technical, aimed at researches wanting to get under the hood and expand / develop.

None of the above is particularly strong suggestions, just ideas!

**Comment by seabbs** [comment]
Yes, agree with all suggestions. I like the split between estimate users, researchers, and developers - this is probably a good mindset to have for all docs.

I am on board with `roxygen` documentation and this could be linked up into a `pkgdown` site or similar (by dummying an R package). Could cause a little headache but might be a nice way to give people rendered function by function documentation and help with #49. 

For previous papers, I have done this as a way of surfacing individual model steps etc and it worked out quite well (though whether anyone else ever used it is a question open for debate).
